recipes/scnlib/all/conanfile.py

```diff
@@ -93,12 +106,22 @@ def package(self):
         self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
         if self.options.header_only:
             self.copy("*", dst="include", src=os.path.join(self._source_subfolder, "include"))
-            self.copy("*", dst=os.path.join("include", "scn", "detail"), src=os.path.join(self._source_subfolder, "src"))
+            if tools.Version(self.version) >= "1.0":
+                self.copy("reader_*.cpp", dst=os.path.join("include", "scn", "reader"), src=os.path.join(self._source_subfolder, "src"))
+                self.copy("vscan.cpp", dst=os.path.join("include", "scn", "scan"), src=os.path.join(self._source_subfolder, "src"))
+                self.copy("locale.cpp", dst=os.path.join("include", "scn", "detail"), src=os.path.join(self._source_subfolder, "src"))
+                self.copy("file.cpp", dst=os.path.join("include", "scn", "detail"), src=os.path.join(self._source_subfolder, "src"))
+            else:
+                self.copy("*.cpp", dst=os.path.join("include", "scn", "detail"), src=os.path.join(self._source_subfolder, "src"))
```

Comment:
I'm lot, what's the meaning of header_only here for 1.0 if you have to package .cpp files? If it's not supported in 1.0, just remove header_only option.

Reply:
scnlib includes .cpp files directly in header only mode.
That's is why this recipe packages .cpp files.

For example:
https://github.com/eliaskosunen/scnlib/blob/v1.0/include/scn/reader/int.h#L449-L451